tests/test_amqps.py

```diff
@@ -0,0 +1,48 @@
+import ssl
+from functools import partial
+
+import pytest
+
+import aio_pika
+from tests.test_amqp import TestCaseAmqp
+
+
+@pytest.fixture(
+    scope="module", params=[aio_pika.connect, aio_pika.connect_robust],
+)
+def connection_fabric(request):
+    return request.param
+
+
+@pytest.fixture
+def create_connection(connection_fabric, loop, amqp_url):
```

Comment:
Didn't find where this fixture is being used..

Reply:
It’s overlapping the conftest fixture